app/openlxp_xds_project/settings.py

```diff
@@ -171,6 +173,50 @@
 
 AUTH_USER_MODEL = 'core.XDSUser'
 
+# openlxp_authentication settings
+SOCIAL_AUTH_STRATEGY = 'openlxp_authentication.models.SAMLDBStrategy'
+JSONFIELD_ENABLED = True
+USER_MODEL = 'core.XDSUser'
+SESSION_EXPIRATION = True
+if(os.environ.get('LOGIN_REDIRECT_URL') != None):
+    LOGIN_REDIRECT_URL = os.environ.get('LOGIN_REDIRECT_URL')
+
+if(os.environ.get('OVERIDE_HOST') != None):
+    OVERIDE_HOST = os.environ.get('OVERIDE_HOST')
+    BAD_HOST = os.environ.get('BAD_HOST')
+
+if(os.environ.get('STRATEGY') != None):
+    SOCIAL_AUTH_STRATEGY = os.environ.get('STRATEGY')
+
+SP_ENTITY_ID = os.environ.get('ENTITY_ID')
+
+SP_PUBLIC_CERT = os.environ.get('SP_PUBLIC_CERT')
+SP_PRIVATE_KEY = os.environ.get('SP_PRIVATE_KEY')
+ORG_INFO = {
```

Comment:
It might be useful to future people to put a link to the attribute definitions for these config blocks in a code comment.  Also I believe there are a few more attribute sets that can be specified as well.

https://github.com/onelogin/python-saml/blob/master/README.md

Reply:
Yeah, I can add a link to the authentication package's README and the docs for social-django.  I don't think we should link to onelogin's python3-saml documentation, since any settings would be configured through social-django's or our authentication package's settings